spec/features/admin/password_reset_spec.rb

```diff
@@ -6,19 +6,37 @@
     ActionMailer::Base.default_url_options[:host] = 'http://example.com'
   end
 
-  scenario 'allows a user to supply an email for the password reset' do
-    user = create(:user, email: 'foobar@example.com', password: 'secret', password_confirmation: 'secret')
-    visit spree.admin_login_path
-    click_link 'Forgot Password?'
-    fill_in 'Email', with: 'foobar@example.com'
-    click_button 'Reset my password'
-    expect(page).to have_text 'You will receive an email with instructions'
+  context 'when the an account with this email address exists' do
+    before { user = create(:user, email: 'foobar@example.com', password: 'secret', password_confirmation: 'secret') }
```

Comment:
I think the variable assignent `user = ` can be removed ✂️

Reply:
I would prefer a `let!(:user)` instead.